### PR TITLE
Drop yarn

### LINKED ISSUE #1192: Switch from Yarn to NPM
Since NPM is the default package manager for Node and in the recent versions it isn't better than Yarn, we can switch back to it.

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Closes #1192

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
   },
   "homepage": "http://getsaleor.com/",
   "engines": {
-    "node": "6.x",
-    "yarn": "1.0.2"
+    "node": "6.x"
```
Comment (patrys) [inline package.json]: Shouldn't we require Node 8 and NPM 5?

Comment (dominik-zeglen) [reply package.json]: You are totally right

**Comment by patrys** [comment]
Do we need to update the Circle config file? Heroku config?

**Comment by dominik-zeglen** [comment]
Yeah we should update CircleCI config, totally forgot about it.

**Comment by dominik-zeglen** [comment]
@elwoodxblues sure, working on it.

**Pull request merged**